apps/frontend/src/modules/users/slices/actions.ts

```diff
@@ -9,10 +9,16 @@ const loadAll = createAsyncThunk<
 	UserGetAllResponseDto,
 	undefined,
 	AsyncThunkConfig
->(`${sliceName}/load-all`, (_, { extra }) => {
+>(`${sliceName}/load-all`, async (_, { extra, rejectWithValue }) => {
 	const { userApi } = extra;
 
-	return userApi.getAll();
+	try {
+		return await userApi.getAll();
+	} catch (error) {
```

Comment:
you do not need to catch it here. your middleware will do it

Reply:
Got it